### PR TITLE
Add Colab Tutorial

### PR DESCRIPTION
**Pull request by zytx121**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Add Colab Tutorial.

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Review by yangxue0827 — APPROVED** [review]
LGTM

**Review by jbwang1997 — APPROVED** [review]
LGTM

**Comment by ZwwWayne** [comment]
1. we should not use dev branch but should recommend users to use the released master branch
2. suggest renaming it as inference_demo.ipynb

**Comment by ZwwWayne** [comment]
Should we also teach users how to train the model on a new dataset?

**Comment by zytx121** [comment]
> 1. we should not use dev branch but should recommend users to use the released master branch
> 2. suggest renaming it as inference_demo.ipynb

There is a bug in the master branch that will cause an error when running the demo. This bug has been fixed in the dev branch. So currently only the dev branch can be used.

**Comment by zytx121** [comment]
> Should we also teach users how to train the model on a new dataset?

OK, I will try it.

**Comment by jbwang1997** [comment]
image
The downloading command has been commented.

File: README_zh-CN.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ https://user-images.githubusercontent.com/10410257/154433305-416d129b-60c8-44c7-
 ## 教程
 
 请参考 [get_started.md](docs/zh_cn/get_started.md) 了解 MMRotate 的基本使用。
+我们为新手提供了 [colab 教程](demo/inference_demo.ipynb)。
```
Comment (jbwang1997) [inline README_zh-CN.md:69]: 新手 -> 初学者

Comment (zytx121) [reply README_zh-CN.md:69]: Done.

**Comment by zytx121** [comment]
> 1. we should not use dev branch but should recommend users to use the released master branch
> 2. suggest renaming it as inference_demo.ipynb

Since we added a section about how to train the model on a new dataset, should we rename it or not?

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**